inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/extract_image_patches.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2020 Intel Corporation
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
Why have you changed the copyright here?

Reply:
It was changed by the commit a748c26fee8858eea4d62d37a09bd926ab1c4ce0 merged to master, and I updated it to avoid merge conflict.

https://github.com/openvinotoolkit/openvino/blob/a748c26fee8858eea4d62d37a09bd926ab1c4ce0/inference-engine/src/mkldnn_plugin/nodes/extract_image_patches.cpp#L1

Reply:
Should I change it back to 2020?